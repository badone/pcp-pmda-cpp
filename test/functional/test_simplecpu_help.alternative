
Usage: ./pmdasimplecpu [options]

Libpcp-pmda options:
  -D [ --debug ] [=spec(=-1)]           set debug specification
  -d [ --domain ] n (=129)              domain number to use
  -h [ --help-file ] file (=/var/lib/pcp/pmdas/simplecpu/help)
                                        file containing help text
  -i [ --inet ] port                    use inet port for pmcd comms; conflicts
                                        with -p, -u and -6
  -l [ --log-file ] file (=simplecpu.log)
                                        file to use for logging
  -p [ --pipe ]                         use stdin/stdout for pmcd comms; 
                                        conflicts with -i, -u and -6
  -u [ --unix ] socket                  use named socket for pmcd comms; 
                                        conflicts with -i, -p and -6

Extra options:
  --config file (=/var/lib/pcp/pmdas/simplecpu/config)
                                        load options from config file
  --export-all [=dir(=.)]               export domain, help, pmns and root then
                                        exit
  --export-domain [=file(=-)]           export domain header then exit
  --export-help [=file(=-)]             export help text then exit
  --export-pmns [=file(=-)]             export pmns text then exit
  --export-root [=file(=-)]             export pmns root then exit
  --help                                display this message then exit
  --version                             display version info then exit

