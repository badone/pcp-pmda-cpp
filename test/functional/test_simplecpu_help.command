# -6 [ --inet6 ] only added for PCP 3.8.1+
pmdasimplecpu --help | grep -Eve 'inet6|with -i, -p and -u'
