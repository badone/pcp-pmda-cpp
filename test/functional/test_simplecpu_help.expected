
Usage: pmdasimplecpu [options]

Libpcp-pmda options:
  -D [ --debug ] [=arg(=-1)]            set debug specification
  -d [ --domain ] arg (=129)            domain number to use
  -h [ --help-file ] arg (=/var/lib/pcp/pmdas/simplecpu/help)
                                        file containing help text
  -i [ --inet ] arg                     use inet port for pmcd comms; conflicts
                                        with -p, -u and -6
  -l [ --log-file ] arg (=simplecpu.log)
                                        file to use for logging
  -p [ --pipe ]                         use stdin/stdout for pmcd comms; 
                                        conflicts with -i, -u and -6
  -u [ --unix ] arg                     use named socket for pmcd comms; 
                                        conflicts with -i, -p and -6

Extra options:
  --config arg (=/var/lib/pcp/pmdas/simplecpu/config)
                                        load options from config file
  --export-all [=arg(=.)]               export domain, help, pmns and root then
                                        exit
  --export-domain [=arg(=-)]            export domain header then exit
  --export-help [=arg(=-)]              export help text then exit
  --export-pmns [=arg(=-)]              export pmns text then exit
  --export-root [=arg(=-)]              export pmns root then exit
  --help                                display this message then exit
  --version                             display version info then exit

